lib/rules/declaration-block-no-redundant-longhand-properties/__tests__/index.mjs

```diff
@@ -692,6 +692,34 @@ testRule({
 			description: 'explicit grid-row test',
 			message: messages.expected('grid-row'),
 		},
+		{
+			code: stripIndent`
+				a {
+					grid-row-start: 1;
+					grid-row-end: 2;
+					grid-column-start: 3;
+					grid-column-end: 4;
+				}
+			`,
+			fixed: stripIndent`
+				a {
+					grid-row: 1 / 2;
+					grid-column: 3 / 4;
+				}
+			`,
+			description: 'explicit grid-area test',
+			warnings: [
+				{ column: 2, endColumn: 14, endLine: 3, line: 3, message: messages.expected('grid-row') },
+				{ column: 2, endColumn: 17, endLine: 5, line: 5, message: messages.expected('grid-area') },
```

Comment:
question

Should it be

```
				{ column: 2, endColumn: 17, endLine: 5, line: 5, message: messages.expected('grid-column') },
```

instead?

Reply:
I also agree it's weird, but it's out of the scope of this PR/issue. And I want to keep the scope to a minimum for gradual improvements.

If you're interested, can you open a new issue?

Reply:
So you are saying it's not a consequence of this PR and the warning was reported before.

Reply:
Right. As you know, this rule is very hard to implement *correctly* 😅

Reply:
Then let's at least remove the description.

Reply:
I've fixed it.
[fix: update description for grid-row and grid-column test](https://github.com/stylelint/stylelint/pull/8023/commits/65b3d089b6fc360a85d327358652a839eeed654c)